core/modules/layout/layout.api.php

```diff
@@ -719,8 +719,8 @@ function hook_block_view_alter(&$data, $block) {
  *   - module: The name of the module that defined the block.
  *   - delta: The unique identifier for the block within that module, as defined
  *     in hook_block_info().
- * @param array $settings
- *   An array of settings for this block.
+ *   - settings: All block settings as defined for this instance of the block.
+ *   - contexts: All layout contexts available for the layout.
```

Comment:
Both, settings and contexts are arrays - maybe also mention that?

Reply:
OK, done